static/app/views/dashboards/widgetCard/metricWidgetCard/index.tsx

```diff
@@ -83,9 +83,13 @@ export function MetricWidgetCard({
       toMetricDisplayType(metricWidgetQueryParams.displayType)
     );
 
+    const isCustomTitle = parseField(title) === null && title !== '';
```

Comment:
k, there is this tiny corner case where it breaks if I enter a valid field, right?

Reply:
Could we do something like this?
```suggestion
    const isCustomTitle = title !== '' &&
      parseField(title)?.mri !== metricWidgetQueryParams.mri;
```